pybotters/models/gmocoin.py

```diff
@@ -0,0 +1,616 @@
+    def to_position(data: Item) -> "Position":
+        return Position(
+            position_id=data["positionId"],
+            symbol=Symbol[data["symbol"]],
+            side=OrderSide[data["side"]],
+            size=Decimal(data["size"]),
+            orderd_size=Decimal(data["orderdSize"]),
+            price=int(data["price"]),
+            loss_gain=int(data["lossGain"]),
+            leverage=Decimal(data["leverage"]),
+            losscut_price=int(data["losscutPrice"]),
+            timestamp=parse_datetime(data.get("timestamp")),
+        )
+
+    @staticmethod
+    def to_position_summaries(data: List[Item]) -> List["PositionSummary"]:
+        return [MessageHelper.to_position_summary(x) for x in data]
+
+    @staticmethod
+    def to_position_summary(data: Item) -> "PositionSummary":
+        return PositionSummary(
+            symbol=Symbol[data["symbol"]],
+            side=OrderSide[data["side"]],
+            average_position_rate=int(data["averagePositionRate"]),
+            position_loss_gain=int(data["positionLossGain"]),
+            sum_order_quantity=Decimal(data["sumOrderQuantity"]),
+            sum_position_quantity=Decimal(data["sumPositionQuantity"]),
+            timestamp=parse_datetime(data.get("timestamp"))
+            if data.get("timestamp")
+            else datetime.now(timezone.utc),
+        )
+
+
+class GmoCoinDataStoreInterface(DataStoreInterface):
+    def _init(self) -> None:
+        self.create("orderbook", datastore_class=OrderBookStore)
+        self.create("orders", datastore_class=OrderStore)
+        self.create("positions", datastore_class=PositionStore)
+        self.create("executions", datastore_class=ExecutionStore)
+        self.create("position_summary", datastore_class=PositionSummaryStore)
+
+    async def initialize(self, *aws: Awaitable[aiohttp.ClientResponse]) -> None:
+        for f in asyncio.as_completed(aws):
+            resp = await f
+            data = await resp.json()
+            if (
+                resp.url.path == "/private/v1/latestExecutions"
+                and "list" in data["data"]
+            ):
+                self.executions._onresponse(
+                    MessageHelper.to_executions(data["data"]["list"])
+                )
+            if resp.url.path == "/private/v1/activeOrders" and "list" in data["data"]:
+                self.orders._onresponse(MessageHelper.to_orders(data["data"]["list"]))
+            if resp.url.path == "/private/v1/openPositions" and "list" in data["data"]:
+                self.positions._onresponse(
+                    MessageHelper.to_positions(data["data"]["list"])
+                )
+            if (
+                resp.url.path == "/private/v1/positionSummary"
+                and "list" in data["data"]
+            ):
+                self.position_summary._onresponse(
+                    MessageHelper.to_position_summaries(data["data"]["list"])
+                )
+
+    def _onmessage(self, msg: Item) -> None:
```

Comment:
```suggestion
    def _onmessage(self, msg: Item, ws: ClientWebSocketResponse) -> None:
```
第2引数(ws)を設定してないので `TypeError` でonmessageコールバックが動いていません。